Cargo.toml

```diff
@@ -18,5 +18,18 @@ regex = "^0.2"
 version = "^0.0"
 optional = true
 
+[dependencies.serde]
+version = "1"
+optional = true
+
+[dependencies.serde_derive]
+version = "1"
+optional = true
+
 [badges]
 travis-ci = { repository = "sanpii/todo-txt", branch = "master" }
+
+[features]
+default       = []
+serde-support = ["serde", "serde_derive", "chrono/serde"]
```

Comment:
What is `chrono/serde`?

Reply:
This enables the "serde" feature in the "chrono" dependency when "serde-support" is enabled.

Reply:
We need to test both configurations. You can add a second test command in `.travis.yml` (in a perfect world `cargo test --all-features` covert all features but clippy probably fails).